tasks/togeojson.js

```diff
@@ -57,9 +58,19 @@ module.exports = function gruntTogeojson(grunt) {
         else {
           data = JSON.stringify(geo, null, '  ');
         }
-        grunt.file.write(file.dest, data, {encoding: encoding});
+
+        var dest = file.dest;
+
+        if (options.rename) {
```

Comment:
It might be that I had not done it for this plugin yet, but each option should be checked for the expected type.
Here it would be `typeof options.rename === 'function'`, as oppose to just checking if it exists.